NodeJS-11-12-abonomentai ir useriai/backend/src/index.js

```diff
@@ -0,0 +1,198 @@
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const { MongoClient, ObjectId } = require('mongodb');
+const { check, param, validationResult } = require('express-validator');
+
+const app = express();
+const { PORT } = process.env;
+const { USER } = process.env;
+const { PASS } = process.env;
```

Comment:
Objekto destrukturizacija faina tuo, kad galima viena eilute apsibrezti kelis kintamuosius.
```suggestion
const { PORT, USER, PASS } = process.env;
```